### DISCUSSION TITLE
Playlist Editor Freedom

### DISCUSSION THREAD

**Discussion by BossGamerDK** [efroemling/ballistica#554] [Ideas]
As of right now, created playlist within BombSquad, whether it be in FFA or Teams, are ordered in alphabetical order. Now of course there are probably quite a few people who like it that way of course. Y’know I like it in a way, means I know where everything is. However, I think having the ability to change the order of the Playlist would be a welcome change. I mean we can already do that which each Game inside if a Playlist, so why not have the ability to do that with our Playlist? I already know how I'd order my Playlist if this were to ever become a reality.

**Comment by huayuede** [comment]
我已收到你的来件。

**Reply by MatiasMGz** [reply]
Only English, please.

**Comment by MatiasMGz** [comment]
Yeah, i would like this too. Its very basic and it should be already in the game.
